lib/assets/javascripts/src/local-time/relative_time.coffee

```diff
@@ -70,4 +70,7 @@ class LocalTime.RelativeTime
     strftime(@date, format)
 
   toTimeString: ->
-    strftime(@date, getI18nValue("time.formats.default"))
+    strftime(@date, if @useFormat24
+      getI18nValue("time.formats.default_24h")
+    else
+      getI18nValue("time.formats.default"))
```

Comment:
I think we can do away with all diffs in the controller if we just import the config in this file and do something like:

```suggestion
    format = if config.useFormat24 then "default_24h" else "default"
    strftime @date, getI18nValue("time.formats.#{format}")
```

If that works, let's do that instead.

Reply:
Do you think it should fallback to default if default_24h is missing? This would be easier for other which upgrade the lib.

Reply:
Let's include a test for this nonetheless, to protect us from breaking it in the future

Reply:
> Let's include a test for this nonetheless, to protect us from breaking it in the future

Ah, we already test for this:

https://github.com/basecamp/local_time/blob/1395c561faae447e41da8319b05e6e4bbac294c9/test/javascripts/src/i18n_test.coffee#L35-L49